.goreleaser.yml

```diff
@@ -98,23 +104,31 @@ brews:
     install: |
       bin.install "vcsim"
 dockers:
-- image_templates: 
-  - "vmware/govc:{{ .Tag }}"
-  - "vmware/govc:v{{ .Major }}"
-  - "vmware/govc:v{{ .Major }}.{{ .Minor }}"
-  - "vmware/govc:latest"
-  goos: linux
-  goarch: amd64
-  dockerfile: Dockerfile.govc
-  binaries: 
```

Comment:
I think we should keep `id` (was binaries) to reference the specific binary used in that image (clarity).

Reply:
You are right!  Fixed.

Which also corrects the first concern of `CMD` vs `ENTRYPOINT`.